diff --git a/Engine/Source/Programs/UnrealBuildTool/System/ActionGraph.cs b/Engine/Source/Programs/UnrealBuildTool/System/ActionGraph.cs
index 30cf3e2519c..c82217645cf 100644
--- a/Engine/Source/Programs/UnrealBuildTool/System/ActionGraph.cs
+++ b/Engine/Source/Programs/UnrealBuildTool/System/ActionGraph.cs
@@ -469,23 +469,27 @@ namespace UnrealBuildTool
 				DateTime StartTime = DateTime.UtcNow;
 
 				ActionExecutor Executor;
				if(bIsRemoteCompile)
 				{
 					Executor = new RemoteExecutor();
 				}
				else if(ActionsToExecute.Any(x => x.ActionHandler != null))
 				{
 					Executor = new LocalExecutor();
 				}
+				else if (FASTBuild.IsAvailable())
+				{
+					Executor = new FASTBuild();
+				}
 				else if ((XGE.IsAvailable() && BuildConfiguration.bAllowXGE) || BuildConfiguration.bXGEExport)
 				{
 					Executor = new XGE();
 				}
				else if(BuildConfiguration.bAllowDistcc)
 				{
 					Executor = new Distcc();
 				}
				else if(SNDBS.IsAvailable() && BuildConfiguration.bAllowSNDBS)
 				{
 					Executor = new SNDBS();
 				}
